metadata-ingestion/src/datahub/ingestion/source/sql/postgres.py

```diff
@@ -107,11 +107,17 @@ class PostgresConfig(BasicSQLAlchemyConfig):
 
     database_pattern: AllowDenyPattern = Field(
         default=AllowDenyPattern.allow_all(),
-        description="Regex patterns for databases to filter in ingestion.",
+        description=(
+            "Regex patterns for databases to filter in ingestion. "
+            "Note: this is not used if `database` or `sqlalchemy_uri` are provided."
+        ),
     )
-    database: Optional[str] = Field(
```

Comment:
this field also exists in `BasicSQLAlchemyConfig`, and was only here so we could update the docs for it - imo we still want to include those updated docs

Reply:
Ah I figured it was because we didn't realize the field already existed lol